pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -56,12 +59,16 @@ export const CryptoPolicyRow = () => {
     const Dialogs = useDialogs();
     const [currentCryptoPolicy, setCurrentCryptoPolicy] = useState(null);
     const [fipsEnabled, setFipsEnabled] = useState(null);
+    const [shaSubPolicyAvailable, setShaSubPolicyAvailable] = useState(null);
 
     useEffect(() => {
         cockpit.file("/proc/sys/crypto/fips_enabled").read()
                 .then(content => setFipsEnabled(content ? content.trim() === "1" : false));
         cockpit.file("/etc/crypto-policies/state/current")
                 .watch(content => setCurrentCryptoPolicy(content ? content.trim() : null));
+        // RHEL-8-8 has no SHA1 subpolicy
+        cockpit.file("/usr/share/crypto-policies/policies/modules/SHA1.pmod").read()
+                .then(content => setShaSubPolicyAvailable(content ? content.trim() : false));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18219-20230126-101600-3ffc4e6f-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)